### PR TITLE
ENH: Add post-scanner data flowchart

### LINKED ISSUE #160: Flow chart of the post-scanner steps
- Mostly to summarize the processing steps as well as the transfer of data

### PR DESCRIPTION
**Pull request by acionca**
This PR adds an `intro.md` to the _data-management_ section (similar to the one of _data-collection_) that:
- Details the post-scanner steps to apply to the data data
- Links to the corresponding sections for the conversion to BIDS and processing of the data
- Summarize the data workflows in a flowchart (editable [here](https://mermaid.live/edit#pako:eNqVVu1u2zYUfRWORbAOsAt_tHHmAQNaeUG8pmsapfsxuTAYko6JSCJLUUm8KP_7BHvAPckuScmSbMXbDMMmdQ4vz6F4L_mIqWQcT_Eqlvd0TbRBV-8WKYIPjUmWzfgKiUShzGh5y6cvVoNBz7f794KZ9XSkHn5apH5Ell_faKLW6OJtEEb254sHeMr2OGq9yYSMFvjCNfqaU6mZSG_QOVFGKvTy77---e8PC3wgEDdLRaNfNrxvNKG3XKNAJio3XB-aPdvQtVQbO3SBQyOSPBZIaZ7x1BAjZPp_ZcSSknjJiCHRuW2i0EhNbng5xH40uV8mWkSX5B59uJw3kGvBMge9m8_CLsyulgf9uu3inK08DI0DIpmW6lo-RDP_v6PNTWLF7c1hUTuFBXdmsI3dWWARYsKiGfyfE9bhc1lRDvltk5713abV6uzn6OjfotVckPWVLmVuot9FlsML1FxJbbJGONgfSku6XIHi6MJ3eJZxhlZtDxWR7RGZJ9bU06DZY0Y4CZaGGNfiDjbjHW-KCIPDqy8VT1Oeaxl9hBb6zTaRfRUZb5nJ062q6HPdrqgtNxWvZWWfSCXMTEGwMJsoDHpgruVV83g5jM6CizW65DGH0Wj4pQ2P2vBoBx634fH-XrdlB_X7_Z8LRWiWkBSFVAtliir9PM1vAGR5ZTKgcJPSYpsIZWBbXBzroqwYRbN2lLHqB5aKOiLCtmy-rFKKC3zGcyYCmd6hrBRaZUnNtYJKU9AaWUKbXSsuZ3NsVECrg-3UeHo1l5NyOr8Mr1AGr9cWQJnGG_RdgRJ48UEsUmGrGsThPIUyXa48gG7oAs9TKpitnvHLDEplgUjMtYne2l-k4GQRVCiSVvvFHS-eY0-YSk9dRr1fqkHJr_K6QGcf9efSo2t6h2VWt81Uue6UQSZ9Cor99L7cpnfppCK4UZ--f082btQSnnrXUbhrvon6YUGGUmmQvC1ggoyuOctj3nRcP23a7lRui4rNuOK_FZ7OGGwvxrM1qd6enZHUhvEVoXybT40C4ocdHbXWsE0oU6UhwnF2QjSN7uLVJM4zUkJx2JS8erozso8SaZaUp5nUW3XbJ46g-Y1uqd8-cPBKxHCHqNG9k6STVfWd-NOg05Xd1v6KslzR7XLWBwFYZIcsssqiPS3YrajwsuumLg-SzlW3Ne9qjkgKNfz9vKi5nl12n1v-fRX3wK_qG6jwXQfZG5mpANdxUcPgOV1BOHO6rixXugMNsjDcLk0YuADNg8YDp12Ahxrb0HHcKVQJ2AVGZa424myxMe7hhOuECAY35kfLXGCz5glf4Ck0GdG3C7xIn4BHciNt-cdTo3Pew7mCXcNngoCnBE9XJM7gKWcCbokf_BXc3cR7GGrkH1LWHOjj6SN-wNP-yY-vJsPJyZvjwWjyejSenPTwBk-H48Gr0eTNaDA-Hg8mx8PB66ce_tOFGD79A8wt6lk) if you do not wish to render it)

I believe it is still WIP but I will need @esavary's input for some of the physiological workflow

closes #160

### CODE REVIEW

**Comment by esavary** [comment]
LGTM! If you need more detailed information about the physiological data workflow, apart from the eye-tracking (ET) data, it might be safer to ask @oesteban . I didn't contribute much because we now use the notebook instead of the old code I was trying to use for the pilot sessions. However, I can still try to help if you have specific questions.

**Review by esavary — APPROVED** [review]

File: docs/data-management/intro.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    unprocessed --> rel_1
+    processed --> rel_2
+    connectivity --> rel_3
+```
+
+The above graph can be detailed into two main data workflows:
+
+1. **MRI data**.
+    Stored on the CHUV PACS, the raw MRI data should first be locally [converted to BIDS](post-session.md/#convert-imaging-data-to-bids-with-heudiconv). Then both the raw and BIDS compliant data are to be synchronized with the [*Datalad* dataset](preliminary.md/#adding-data-or-metadata). From there, the BIDS compliant data can be [pre-processed](../processing/preprocessing.md) using the corresponding packages (e.g. *fMRIPrep*, *dMRIPrep*) to allow computation of analysis-grade derivatives (e.g. [functional](../processing/functional-connectivity.md) or structural connectivity).
```
Comment (celprov) [inline docs/data-management/intro.md]:
```suggestion
    Stored on the CHUV PACS, the raw MRI data should first be locally [converted to BIDS](post-session.md/#convert-imaging-data-to-bids-with-heudiconv). Then both the raw and BIDS compliant data are to be synchronized with the [*Datalad* dataset](preliminary.md/#adding-data-or-metadata). From there, the BIDS compliant data can be [quality controlled](./mriqc.md) and [pre-processed](../processing/preprocessing.md) using the corresponding packages (e.g. *MRIQC*, *fMRIPrep*, *dMRIPrep*) to allow computation of analysis-grade derivatives (e.g. [functional](../processing/functional-connectivity.md) or structural connectivity).
```

File: docs/data-management/intro.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    subgraph physio["Physio-recording Laptop ({{ secrets.hosts.acqknowledge | default("███") }})"]
+    end
+
+    subgraph et_pc[Eye-tracker Computer]
+    end
+
+    subgraph psychopy_pc["Stimuli presentation Laptop ({{ secrets.hosts.psychopy | default("███") }})"]
+    end
+
+    subgraph local_data[Local Storage]
```
Comment (celprov) [inline docs/data-management/intro.md]: Is it possible to write the label of the box in bold so it stands out more?

File: docs/data-management/intro.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    bids_mri -->|FIRST session only !| med[Clinical screening]
+    med -->|"Incidental(s)"| alert[Alert participant]
+    class alert imp
+
+    local_data --->|cron Job| HOrUs
+    HOrUs ---> datalad
+    bids_mri_datalad -->|MRIQC| mriqc_out[Visual Reports]
+
+    mriqc_out -->|Q'Kay| mri_qc_screen[Screening]
+    mri_qc_screen -->|QCs not ok| Reschedule
```
Comment (celprov) [inline docs/data-management/intro.md]:
```suggestion
    mri_qc_screen -->|exclusion criteria found| Reschedule
```

File: docs/data-management/intro.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    class Reschedule imp
+
+    bids_mri_datalad -->|fMRIPrep| preproc_fMRI[Preprocessed fMRI]
+    bids_mri_datalad -->|dMRIPrep| preproc_dMRI[Preprocessed dMRI]
+    
+    bids_mri_datalad -->|pydeface Script| unprocessed
```
Comment (celprov) [inline docs/data-management/intro.md]: Neat pick comment but can you make the capital letters consistent? always capital on first words and either always or never capital on subsequent words. Thanks :)

**Review by celprov — CHANGES_REQUESTED** [review]
Thanks! Looks cool

File: docs/data-management/intro.md (hunk 3 above)
Comment (oesteban) [reply docs/data-management/intro.md]:
```suggestion
    mri_qc_screen -->|exclusion criterium met| Reschedule
```

**Review by oesteban — APPROVED** [review]
LGTM

File: docs/data-management/intro.md (hunk 4 above)
Comment (acionca) [reply docs/data-management/intro.md]: I wanted to keep the scripts name as they are but I'll double check

File: docs/data-management/intro.md (hunk 2 above)
Comment (acionca) [reply docs/data-management/intro.md]: I tried but the markdown formating is messy with mermaid (especially when rendered through mkdocs), I'll investigate

Comment (acionca) [reply docs/data-management/intro.md]: It works now, you should use html formatting such as `<b>bold</b> and <i>italic</i>` to render properly.

File: docs/data-management/intro.md (hunk 4 above)
Comment (acionca) [reply docs/data-management/intro.md]: Added italic for script names and fixed inconsistencies

**Review by celprov — APPROVED** [review]
Top! Looks more easily readable :)

**Pull request merged**